failed to resolve: could not find `EventsLoop` in `glutin`

Hi! Rust newbie here. I'm trying to run the "Getting Started" section of https://github.com/glium/glium/blob/master/book/tuto-01-getting-started.md with the following code (copied directly from the tutorial):

`src/main.rs`:
```rust
fn main() {
    use glium::{glutin, Surface};

    let mut events_loop = glium::glutin::EventsLoop::new();
    let wb = glium::glutin::WindowBuilder::new();
    let cb = glium::glutin::ContextBuilder::new();
    let display = glium::Display::new(wb, cb, &events_loop).unwrap();

    let mut closed = false;
    while !closed {
        let mut target = display.draw();
        target.clear_color(0.0, 0.0, 1.0, 1.0);
        target.finish().unwrap();

        events_loop.poll_events(|ev| {
            match ev {
                glutin::Event::WindowEvent { event, .. } => match event {
                    glutin::WindowEvent::CloseRequested => closed = true,
                    _ => (),
                },
                _ => (),
            }
        });
    }
}
```

`Cargo.toml`:
```toml
[package]
name = "glium_test"
version = "0.1.0"
authors = ["chrisian"]
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
glium="*"
```

And running into the following issue

```
PS C:\Users\christian\Programming\glium_test> cargo run
    Updating crates.io index
   Compiling cfg-if v0.1.10    
   Compiling log v0.4.8        
   Compiling khronos_api v3.1.0
   Compiling xml-rs v0.8.0     
   Compiling winapi v0.3.8     
   Compiling cc v1.0.50        
   Compiling libc v0.2.67      
   Compiling semver-parser v0.7.0
   Compiling scopeguard v1.1.0   
   Compiling smallvec v1.2.0     
   Compiling bitflags v1.2.1     
   Compiling instant v0.1.2
   Compiling lazy_static v1.4.0
   Compiling rustc-demangle v0.1.16
   Compiling takeable-option v0.5.0
   Compiling fnv v1.0.6
   Compiling lock_api v0.3.3
   Compiling semver v0.9.0
   Compiling rustc_version v0.2.3
   Compiling raw-window-handle v0.3.3
   Compiling gl_generator v0.13.1
   Compiling gl_generator v0.14.0
   Compiling memoffset v0.5.3
   Compiling libloading v0.5.2    
   Compiling backtrace-sys v0.1.32
   Compiling backtrace v0.3.44                                                                                                                 
   Compiling glutin_egl_sys v0.1.4                                                                                                             
   Compiling glutin_wgl_sys v0.1.4
   Compiling glium v0.26.0                                                                                                                     
   Compiling parking_lot_core v0.7.0                                                                                                           
   Compiling parking_lot v0.10.0                                                                                                               
   Compiling winit v0.21.0                                                                                                                     
   Compiling glutin v0.23.0
   Compiling glium_test v0.1.0 (C:\Users\christian\Programming\glium_test)
error[E0433]: failed to resolve: could not find `EventsLoop` in `glutin`
 --> src\main.rs:4:42
  |
4 |     let mut events_loop = glium::glutin::EventsLoop::new();
  |                                          ^^^^^^^^^^ could not find `EventsLoop` in `glutin`

error[E0433]: failed to resolve: could not find `WindowBuilder` in `glutin`
 --> src\main.rs:5:29
  |
5 |     let wb = glium::glutin::WindowBuilder::new();
  |                             ^^^^^^^^^^^^^ could not find `WindowBuilder` in `glutin`

error[E0433]: failed to resolve: could not find `Event` in `glutin`
  --> src\main.rs:17:25
   |
17 |                 glutin::Event::WindowEvent { event, .. } => match event {
   |                         ^^^^^ could not find `Event` in `glutin`

error[E0433]: failed to resolve: could not find `WindowEvent` in `glutin`
  --> src\main.rs:18:29
   |
18 |                     glutin::WindowEvent::CloseRequested => closed = true,
   |                             ^^^^^^^^^^^ could not find `WindowEvent` in `glutin`

error[E0599]: no method named `poll_events` found for type `winit::event_loop::EventLoop<_>` in the current scope
  --> src\main.rs:15:21
   |
15 |         events_loop.poll_events(|ev| {
   |                     ^^^^^^^^^^^ method not found in `winit::event_loop::EventLoop<_>`

error: aborting due to 5 previous errors

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `glium_test`.

To learn more, run the command again with --verbose.
```

Google gives me no results (with the exception of https://github.com/iceiix/stevenarella/issues/115#issuecomment-494988712), and I'm at a bit of a loss for what to do. Is there some setup detail I've missed?

For reference:
```
PS C:\Users\christian\Programming\glium_test> rustc --version --verbose
rustc 1.41.0 (5e1a79984 2020-01-27)
binary: rustc
commit-hash: 5e1a799842ba6ed4a57e91f7ab9435947482f7d8
commit-date: 2020-01-27
host: x86_64-pc-windows-msvc
release: 1.41.0
LLVM version: 9.0
```

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -15,10 +15,10 @@ extern crate glium;
 
 fn main() {
     // 1. The **winit::EventsLoop** for handling events.
-    let mut events_loop = glium::glutin::EventsLoop::new();
+    let mut events_loop = glium::glutin::event_loop::EventLoop::new();
     // 2. Parameters for building the Window.
-    let wb = glium::glutin::WindowBuilder::new()
-        .with_dimensions(1024, 768)
+    let wb = glium::glutin::window::WindowBuilder::new()
+        .with_inner_size(glium::glutin::dpi::LogicalSize::new(1024.0, 768.0))
         .with_title("Hello world");
     // 3. Parameters for building the OpenGL context.
     let cb = glium::glutin::ContextBuilder::new();

